mijia/src/lib.rs

```diff
@@ -138,6 +140,22 @@ impl MijiaEvent {
                 id,
                 event: DeviceEvent::Connected { connected: false },
             } => Some(MijiaEvent::Disconnected { id }),
+            BluetoothEvent::Device {
+                id,
+                event: DeviceEvent::Discovered,
+            } => {
+                // Only pass through discovery events for sensors we recognise.
+                let device = session
+                    .get_device_info(&id)
+                    .await
+                    .map_err(|e| log::error!("Error getting device info: {:?}", e))
+                    .ok()?;
+                if device.name.as_deref() == Some(MIJIA_NAME) {
```

Comment:
(optional) I might be tempted to write a `is_mijia(device)` helper in case we manage to get the service property method working reliably again in the future.

Reply:
Done.